force-app/main/default/classes/LeadTriggerHandler.cls

```diff
@@ -0,0 +1,33 @@
+public class LeadTriggerHandler{
+    public static void createLeadTask(List<Lead> leadRecords, Map<Id, Lead> oldLeadRecords){
+        List<Task> insertTasks = new List<Task>();
+        if(leadRecords.size() > 0){
+            for(Lead leadRecord : LeadRecords){
+                if(trigger.isInsert && leadRecord.ProductInterest__c == 'Cookbook Printing'){
+                    insertTasks.add(leadTriggerHandler.taskMethod('New Cookbook Printing Inquiry', leadRecord.Id, 'New Printing Inquiry'));
```

Comment:
Since you are calling a helper/utility method in the same class, you don't need to reference the class name. You can just call `taskMethod(_parameters_)`.